### PR TITLE
changelog v0.5.0

### PR DESCRIPTION
**Pull request by tlambert03**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
 
-[Full Changelog](https://github.com/napari/npe2/compare/v0.4.0...0.4.1)
+[Full Changelog](https://github.com/napari/npe2/compare/v0.4.1...0.5.0)
+
+**Implemented enhancements:**
+
+- allow `npe2 parse` to output to file, add format option [\#183](https://github.com/napari/npe2/pull/183) ([tlambert03](https://github.com/tlambert03))
+- Add npe1\_shim field to schema [\#182](https://github.com/napari/npe2/pull/182) ([tlambert03](https://github.com/tlambert03))
+- allow `npe2.write` to take layer instances [\#181](https://github.com/napari/npe2/pull/181) ([tlambert03](https://github.com/tlambert03))
+- add npe2pm TestPluginManager fixture [\#180](https://github.com/napari/npe2/pull/180) ([tlambert03](https://github.com/tlambert03))
```
Comment (DragaDoncila) [inline CHANGELOG.md:12]:
```suggestion
- add `npe2pm` `TestPluginManager` fixture [\#180](https://github.com/napari/npe2/pull/180) ([tlambert03](https://github.com/tlambert03))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
-## [0.4.1](https://github.com/napari/npe2/tree/0.4.1) (2022-06-13)
+## [0.5.0](https://github.com/napari/npe2/tree/0.5.0) (2022-06-21)
 
-[Full Changelog](https://github.com/napari/npe2/compare/v0.4.0...0.4.1)
+[Full Changelog](https://github.com/napari/npe2/compare/v0.4.1...0.5.0)
+
+**Implemented enhancements:**
+
+- allow `npe2 parse` to output to file, add format option [\#183](https://github.com/napari/npe2/pull/183) ([tlambert03](https://github.com/tlambert03))
+- Add npe1\_shim field to schema [\#182](https://github.com/napari/npe2/pull/182) ([tlambert03](https://github.com/tlambert03))
```
Comment (DragaDoncila) [inline CHANGELOG.md:10]:
```suggestion
- Add `npe1_shim` field to schema [\#182](https://github.com/napari/npe2/pull/182) ([tlambert03](https://github.com/tlambert03))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+- allow `npe2.write` to take layer instances [\#181](https://github.com/napari/npe2/pull/181) ([tlambert03](https://github.com/tlambert03))
+- add npe2pm TestPluginManager fixture [\#180](https://github.com/napari/npe2/pull/180) ([tlambert03](https://github.com/tlambert03))
+
+**Fixed bugs:**
+
+- Fail silently when caching throws OSError [\#184](https://github.com/napari/npe2/pull/184) ([DragaDoncila](https://github.com/DragaDoncila))
```
Comment (DragaDoncila) [inline CHANGELOG.md:16]:
```suggestion
- Fail silently when caching throws `OSError` [\#184](https://github.com/napari/npe2/pull/184) ([DragaDoncila](https://github.com/DragaDoncila))
```

**Review by DragaDoncila — APPROVED** [review]
Minor nitpicks... Looks good! :tada:

**Comment by tlambert03** [comment]
Thanks, will apply these (they are pulled from the GitHub issues, so I will rename the correspondening PR titles)

**Pull request merged**